index.js

```diff
@@ -0,0 +1,94 @@
+import Book from './modules/book.js';
+import Store from './modules/store.js';
+import Switch from './modules/switch.js';
+import { DateTime } from './modules/luxon.js';
+// UI Class: Handle UI Tasks
+
+class UI {
+  static displayBooks() {
+    const books = Store.getBooks();
+
+    books.forEach((book) => UI.addBookToList(book));
+  }
+
+  static addBookToList(book) {
+    const list = document.querySelector('.book-list');
+
+    const div = document.createElement('div');
+    div.classList.add('book-item');
+
+    div.innerHTML = `
+      <p class='book-p1'>"${book.title}"</p>
+      <p class='book-p1'>by</p>
+      <p class='book-p1'>${book.author}</p>
+      <button class='delete'>Remove</button>
+    `;
+
+    list.appendChild(div);
+  }
```

Comment:
- [ ] As part of the guidelines and ES6 best practice, kindly change your functions with arrow functions. Arrow functions allow us to write shorter function syntax: [for more info](https://www.w3schools.com/js/js_arrow_function.asp)

`let myFunction = (a, b) => a * b;`
### Please the same with all your other functions🙏🙂